Src/coffeescript/cql-execution/test/datatypes/datetime.coffee

```diff
@@ -435,6 +444,59 @@ describe 'DateTime.differenceBetween', ->
     a.differenceBetween(b, DateTime.Unit.SECOND).should.eql new Uncertainty(0)
     a.differenceBetween(b, DateTime.Unit.MILLISECOND).should.eql new Uncertainty(0)
 
+  # NOTE: Skipped for the purposes of Bonnie 11/1 release. Needs to be addressed
+  # afterwards.
+  it.skip 'should handle crossing DST in the spring', ->
+    # NOTE: Since we "spring ahead" the 2nd Sunday of March at 2:00am,
+    # the difference between 1:00am EST and 3:00am EDT is only 1 hour!
+    a = DateTime.parse '2017-03-12T01:00:00.0-05:00'
```

Comment:
These tests cover the cases of going from EST to EDT and back.  It might be good to add tests for another DST transition (CST-CDT, MST->MDT, PST->PDT, or AKST->AKDT), or perhaps all of them.

Reply:
Good idea, but probably a task for after this UTC focused release.